pkg/api/admin.go

```diff
@@ -26,7 +26,19 @@ import (
 // 401: unauthorisedError
 // 403: forbiddenError
 func (hs *HTTPServer) AdminGetSettings(c *contextmodel.ReqContext) response.Response {
-	settings, err := hs.getAuthorizedSettings(c.Req.Context(), c.SignedInUser, hs.SettingsProvider.Current())
+	isVerbose := false
+	if _, disableAutoLogin := c.Req.URL.Query()["verbose"]; disableAutoLogin {
```

Comment:
I don't think it's `disableAutoLogin` you want :slightly_smiling_face:

Reply:
Removed, since I had to separate this in 2 different endpoints.